<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="delAction"/>
    <addaction name="separator"/>
    <addaction name="checkAllAction"/>
    <addaction name="dateTimeAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="wordWrapAction"/>
    <addaction name="fontAction"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_Z">
     <property name="title">
      <string>缩放(&amp;Z)</string>
     </property>
     <addaction name="zoomInAction"/>
     <addaction name="zoomOutAction"/>
     <addaction name="restoreZoomAction"/>
    </widget>
    <addaction name="menu_Z"/>
    <addaction name="statusBarAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="viewHelpAction"/>
    <addaction name="separator"/>
    <addaction name="aboutNotepadaction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>asdasf</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="newAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建(N)       CTRL+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>保存(S)        CTRL+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="delAction">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="checkAllAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="dateTimeAction">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="wordWrapAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="statusBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="viewHelpAction">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="aboutNotepadaction">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="restoreZoomAction">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>关于QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
